packages/mermaid/src/diagrams/treeView/db.ts

```diff
@@ -0,0 +1,72 @@
+import type { TreeViewDiagramConfig } from '../../config.type.js';
+import type { TreeViewDb, Node } from './types.js';
+import { getConfig as getCommonConfig } from '../../config.js';
+import DEFAULT_CONFIG from '../../defaultConfig.js';
+import {
+  getAccDescription,
+  getAccTitle,
+  getDiagramTitle,
+  setAccDescription,
+  setAccTitle,
+  setDiagramTitle,
+} from '../common/commonDb.js';
+import { cleanAndMerge } from '../../utils.js';
+let cnt = 0;
+
+const auxRoot: Node = {
+  id: cnt++,
+  level: -1,
+  name: '/',
+  children: [],
+};
+
+let stack = [auxRoot];
+
+const clear = () => {
+  cnt = 0;
+  stack = [auxRoot];
+  auxRoot.children = [];
+  auxRoot.id = cnt++;
+};
```

Comment:
Please use [ImperativeState](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/utils/imperativeState.ts) to store state data.